tools/lib/provision.py

```diff
@@ -128,6 +128,17 @@
 family = distro_info['DISTRIB_FAMILY']
 if not (vendor in SUPPORTED_PLATFORMS and codename in SUPPORTED_PLATFORMS[vendor]):
     logging.critical("Unsupported platform: {} {}".format(vendor, codename))
+    if codename == 'trusty':
+        print()
+        print("Ubuntu Trusty reached end-of-life upstream and is no longer a supported platform for Zulip")
+        try:
+            # This is a bit of a hack to check for Vagrant, as it's possible (although uncommon) to
+            # connect to a Vagrant box as a different user.
+            if subprocess.check_call(["grep", "-q", "^vagrant:", "/etc/passwd", ]) == 0:
```

Comment:
This throws an exception if there’s no match. You meant `subprocess.call`. But how about `os.path.exists('/home/vagrant')` instead?